protocol/synthetix/contracts/storage/Pool.sol

```diff
@@ -414,11 +417,10 @@ library Pool {
         );
         int256 getPositionDebtD18 = updateAccountDebt(self, collateralType, accountId);
 
-        // if they have a credit, just treat their debt as 0
         return
-            getPositionCollateralValueD18.divDecimal(
-                getPositionDebtD18 < 0 ? 0 : getPositionDebtD18.toUint()
-            );
+            getPositionDebtD18 > 0
+                ? getPositionCollateralValueD18.divDecimal(getPositionDebtD18.toUint())
+                : 1e20;
```

Comment:
Just FYI, I am working on another PR that returns Max uint 256 here in the case of infinite cratio, and does so earlier in this code block to avoid getting the collateral value. I think we can merge this and I'll update the other PR.

Reply:
:+1: